backends/opt/ceed-opt-operator.c

```diff
@@ -248,6 +248,10 @@ static inline int CeedOperatorSetupInputs_Opt(CeedInt num_input_fields,
           CeedChkBackend(ierr);
           impl->input_states[i] = state;
         }
+        // Get evec
+        ierr = CeedVectorGetArrayRead(impl->e_vecs_full[i], CEED_MEM_HOST,
```

Comment:
This PR caught a benign misuse where we were requesting Read access to a vector with no data. This had no practical impact since we never used this array pointer, but it is a nice bug to have fixed.